features/picture.yml

```diff
@@ -1,9 +1,9 @@
-draft_date: 2024-06-05
 name: <picture>
-description: The `<picture>` element TODO.
+description: The `<picture>` element is used for responsive images.
```

Comment:
I think needs to be elaborated a little bit. https://web.dev/learn/design/picture-element looks like a good intro to the element, maybe a description can be condensed from that.

Reply:
Updated